app/views/documents/index.html.erb

```diff
@@ -1,57 +1,145 @@
+  } %>
 <% end %>
 
-<h1 class="page-header"><%= current_format.title.pluralize %></h1>
+<% content_for :title do %>
+  <%= "#{current_format.title} finder" %>
+<% end %>
 
-<div class="row">
-  <div class="col-md-12 text-right">
-    <p>
-      <% if current_format.target_stack == "draft" %>
-        <%= link_to "Preview draft (opens in new tab)", current_format.draft_url, target: "_blank" %>
-      <% elsif current_format.target_stack == "live" %>
-        <%= link_to "View on website (opens in new tab)", current_format.live_url, target: "_blank" %>
-      <% end %>
-    </p>
-  </div>
-</div>
+<%= render "govuk_publishing_components/components/inset_text", {} do %>
+  <% if current_format.target_stack == "draft" %>
+    <%= link_to "Preview draft (opens in new tab)", current_format.draft_url, class: "govuk-link", target: "_blank", rel: "noopener" %>
+  <% elsif current_format.target_stack == "live" %>
+    <%= link_to "View on website (opens in new tab)", current_format.live_url, class: "govuk-link", target: "_blank", rel: "noopener" %>
+  <% end %>
+<% end %>
 
-<div class="row">
-  <div class="sidebar col-md-3">
-    <%= link_to "Add another #{current_format.title}", new_document_path(current_format.admin_slug), class: 'action-link' %>
-    <hr>
-    <%= link_to "Finder setup", finder_path(current_format.admin_slug), class: 'action-link' %>
+<%= render "govuk_publishing_components/components/secondary_navigation", {
+  aria_label: "Finder navigation",
+  items: [
+    {
+      label: "Finder documents",
+      href: documents_path(params[:document_type_slug]),
+      current: true,
+    },
+    {
+      label: "Finder setup",
+      href: finder_path(current_format.admin_slug),
+    },
+  ]
+} %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-third">
+    <%= form_tag(documents_path(current_format.admin_slug), method: :get, class: "govuk-!-padding-3 bg-light-grey") do %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Filter by",
+        margin_bottom: 6,
+      } %>
 
-    <%= form_tag(documents_path(current_format.admin_slug), method: :get, class: "add-vertical-margins well") do %>
       <% if current_format.has_organisations? %>
-        <%= label_tag "organisation", "Organisation" %>
-        <%= select_tag "organisation",
-              options_for_select(organisations_options_with_all, selected_organisation_or_current(@organisation)),
-              class: "select2 form-control add-bottom-margin" %>
+        <%= render "govuk_publishing_components/components/select", {
+          id: "filter-organisations",
+          label: "Organisations",
+          options: organisation_options_for_design_system(selected_organisation_or_current(@organisation))
+        } %>
       <% end %>
 
-      <%= label_tag "query", "Search" %>
-      <%= text_field_tag("query", @query, class: "form-control add-bottom-margin") %>
+      <%= render "govuk_publishing_components/components/input", {
```

Comment:
Should we make this a `type="search"`? I believe this is the functionality we have in whitehall.